### PR TITLE
fix deletion of remote token

### PR DESCRIPTION
**Pull request by ntarocco**

### CODE REVIEW

File: invenio_app_ils/patrons/anonymization.py [hunk 1]
```diff
@@ -180,22 +180,28 @@ def delete_user_account(patron_pid):
...
-        ).delete()
-        dropped += RemoteAccount.query.filter(
+        ).delete() or 0
+
+        ra = RemoteAccount.query.filter(
             RemoteAccount.user_id == patron_pid
-        ).delete()
+        ).one_or_none()
+        if ra:
+            dropped += RemoteToken.query.filter(
```
Comment (ntarocco) [inline invenio_app_ils/patrons/anonymization.py:197]: @kprzerwa @FlorianCassayre fixed Sentry exception for RemoteToken

File: invenio_app_ils/circulation/config.py [hunk 2]
```diff
@@ -56,8 +56,6 @@
 ILS_CIRCULATION_MAIL_MSG_CREATOR = (
     "invenio_app_ils.circulation.mail.factory:default_loan_message_creator"
 )
-#: Loan list message creator class
-ILS_CIRCULATION_LOANS_MAIL_MSG_CREATOR = "invenio_app_ils.circulation.mail.factory:default_loan_list_message_creator"
```
Comment (ntarocco) [inline invenio_app_ils/circulation/config.py:60]: @jrcastro2 please double check this clean up, it was all dead code after the moving of active loans email

File: invenio_app_ils/patrons/anonymization.py (hunk 1 above)
Comment (FlorianCassayre) [reply invenio_app_ils/patrons/anonymization.py:197]: Great, thanks :+1:

Comment (kpsherva) [reply invenio_app_ils/patrons/anonymization.py:197]: 👍

**Review by kpsherva — APPROVED** [review]

**Review by jrcastro2 — APPROVED** [review]

**Pull request merged**